ArduCopter/mode_guided.cpp

```diff
@@ -13,9 +13,10 @@
 #define GUIDED_POSVEL_TIMEOUT_MS    3000    // guided mode's position-velocity controller times out after 3seconds with no new updates
 #define GUIDED_ATTITUDE_TIMEOUT_MS  1000    // guided mode's attitude controller times out after 1 second with no new updates
 
-static Vector3f guided_pos_target_cm;       // position target (used by posvel controller only)
-static Vector3f guided_vel_target_cms;      // velocity target (used by velocity controller and posvel controller)
-static uint32_t posvel_update_time_ms;      // system time of last target update to posvel controller (i.e. position and velocity update)
+static Vector3f guided_pos_target_cm;       // position target (used by pos and pos_vel_accel controller)
+static Vector3f guided_vel_target_cms;      // velocity target (used by pos_vel_accel controller and vel_accel controller)
+static Vector3f guided_accel_target_cmss;   // acceleration target (used by pos_vel_accel controller vel_accel controller and accel controller)
+static uint32_t posvel_update_time_ms;      // system time of last target update to pos_vel_accel, vel_accel and accel controller (i.e. position and velocity update)
 static uint32_t vel_update_time_ms;         // system time of last target update to velocity controller
```

Comment:
maybe we should rename:

- vel_update_time_ms to vel**accel**_update_time_ms
- posvel_update_time_ms to posvel**accel**_update_time_ms